Releasing.md

```diff
@@ -0,0 +1,34 @@
+# Release process
+
+Only for Maintainers.
+
+1. Tag with version to be released e.g.:
+
+   ```sh
+   git tag -a 0.4.0-beta -m "0.4.0-beta"
+   git push origin 0.4.0-beta
+   ```
+
+2. Wait for the [Pack and publish to MyGet
+   workflow](https://github.com/open-telemetry/opentelemetry-dotnet/actions?query=workflow%3A%22Pack+and+publish+to+Myget%22)
+   to finish. It's triggered when the you create a new tag. At the end of this,
+   MyGet will have the packages.
+
+3. Validate using MyGet packages. Basic sanity checks :)
```

Comment:
List the commands (like nuget) to do basic sanity check?

Reply:
dont plan to write this, as we intend to have this entire thing automated with no human touch.